source/sampling.txt

```diff
@@ -0,0 +1,55 @@
+.. _sampling:
+
+========
+Sampling
+========
+
+.. default-domain:: mongodb
+
+.. _compass-faq-sampling:
+
+Sampling in |compass| is the selection of a subset of documents from a 
+collection for analysis. Analyzing a sample set of data is a common 
+statistical analysis technique; the results of analyzing a sample set 
+tend to be similar to the results of analyzing an entire data set.
+
+|compass-short| uses sampling for efficiency. Generally, standard 
+sample sets can be selected and analyzed in a few seconds. Analyzing 
+large samples or entire collections demands significantly more time and 
+processing power.
+
+.. _compass_fallback_sampling:
+
+Sampling Method
+---------------
+
+|compass| uses a different approach to sampling depending on your 
+MongoDB instance version.
+
+.. tabs::
+
+   .. tab:: MongoDB 3.2 and later
+      :tabid: sampling-later-version
+
+      In MongoDB 3.2 and later, collections are sampled with the
+      :manual:`$sample </reference/operator/aggregation/sample/>` 
+      operator via the 
+      :manual:`aggregation pipeline </core/aggregation-pipeline>`. This 
+      provides efficient, random sampling without replacement over an 
+      entire collection, or over the subset of documents specified by a 
+      query.
+
+   .. tab:: MongoDB 3.0
```

Comment:
3.6 looks to be the oldest version that Compass supports: https://docs-mongodbcom-staging.corp.mongodb.com/compass/zach.carr/DOCSP-4806/install.html

Consider removing the tabs and the info for MDB 3.0.

Reply:
Great catch, updating the page.